include/sway/tree/container.h

```diff
@@ -63,7 +63,8 @@ struct sway_container {
 	 */
 	size_t id;
 
-	char *name;
+	char *name;            // The view's title (unformatted)
```

Comment:
Nitpick: maybe this should be renamed to `title` for consistency?

Reply:
I kind of agree, but that also means we'd be changing `output->name` to `output->title` and `workspace->name` to `workspace->title`, and I feel like `name` is a better name for those.

Reply:
Yes. you're right.